src/storage/db_interface_frontend.py

```diff
@@ -277,6 +277,28 @@ def find_missing_files(self):
                 parent_to_included.pop(parent_uid)
         return parent_to_included
 
+    def find_orphaned_objects(self):
+        ''' find File Objects whose parent firmware is missing '''
+        orphans_by_parent = {}
+        query_result = list(self.file_objects.aggregate([
```

Comment:
`query_result` does not tell me anything about it's content. Since no one - aside from you, dear Sir - understands aggregate queries, you should rename the variable.

Reply:
renamed to `fo_parent_uids`